database/database.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/itemComparison', {useNewUrlParser: true, useUnifiedTopology: true});
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  console.log('CONNECTED TO DB');
+});
+
+const comparisonSchema = new mongoose.Schema({
+  productId: Number,
```

Comment:
Adding a unique key property to the productId in your schema would help prevent duplicate records when seeding.